courses/machine_learning/deepdive2/how_google_does_ml/solutions/analyze_with_bigquery_solution.ipynb

```diff
@@ -37,7 +37,7 @@
   },
```

Comment:
<p>Explain how the analysis in this section relates to the machine learning problem at hand if it does, otherwise remove it. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/training-data-analyst/pull/615/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='17'/>

Reply:
<p>As stated earlier this is a data analysis effort aimed at supporting the downstream creation of one or many models.</p>